packages/account/src/Components/poi/status/unsupported/detail-component.tsx

```diff
@@ -32,19 +50,19 @@ const DetailComponent = ({
     is_mt5,
     handlePOIforMT5Complete,
     ...props
-}) => {
-    const [status, setStatus] = React.useState();
-    const [response_error, setError] = React.useState();
+}: TDetailComponent) => {
+    const [status, setStatus] = React.useState<string>();
+    const [response_error, setError] = React.useState<string | undefined>('');
```

Comment:
Now as you've added default value, do we need undefined here for types? 
it seems there are no cases for being the value as undefined here
could you please check